help/user-cards.md

```diff
@@ -60,4 +60,5 @@ permission](/help/configure-email-visibility) to view it.
 ## Related articles
 
 - [View someone's profile](/help/view-someones-profile)
+- [Link to a profile](/help/link-to-profile)
```

Comment:
Let's also add the new article to related articles on /help/view-someones-profile.